src/main/java/menion/android/whereyougo/utils/Utils.java

```diff
@@ -186,10 +186,9 @@ public static boolean isAndroid23OrMore() {
         return Utils.parseInt(Build.VERSION.SDK) >= 9;
     }
 
-    /**********************************************/
-  /* OTHERS SECTION */
-
-    /**********************************************/
+    /*
+     * OTHERS SECTION
+    */
```

Comment:
Maybe change to /* OTHERS SECTION */ like in the other classes?

Reply:
Added in the next force push